### PR TITLE
Refactor docker-compose file

### PR DESCRIPTION
**Pull request by PrajwolAmatya**
## Description
In this PR, services that are not required to run dicom-viewer app have been removed. Only required services are included.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -82,7 +37,6 @@ services:
     restart: unless-stopped
     depends_on:
       - traefik
-      - tika-service
 
   selenium:
     build:
```
Comment (saw-jan) [inline docker-compose.yml:42]: we use playwright for e2e so selenium is also not required

File: docker-compose.yml [hunk 2]
```diff
@@ -303,32 +90,12 @@ services:
       traefik.http.middlewares.cors.headers.accesscontrolmaxage: 100
       traefik.http.middlewares.cors.headers.addvaryheader: true
     ports:
-      - "80:80"
       - "8090:8080"
```
Comment (saw-jan) [inline docker-compose.yml:93]:
```suggestion
      - "8080:8080"
```

File: docker-compose.yml [hunk 3]
```diff
@@ -303,32 +90,12 @@ services:
...
-
-  tika:
-    image: apache/tika:2.8.0.0
-    container_name: web_tika
-    ports:
-      - 9998:9998
-    restart: unless-stopped
-
 volumes:
   uploads:
```
Comment (saw-jan) [inline docker-compose.yml:101]: no volumes required if selenium is removed
```suggestion
```

**Review by saw-jan — COMMENTED** [review]
LGTM

CC @koebel

**Review by koebel — APPROVED** [review]
Good that you noticed the unnecessary code blocks. I had just copied the whole thing from another app extension and haven't paid any attention to what is really needed for this app.

**Pull request merged**